prepare-for-release.sh

```diff
@@ -13,7 +13,7 @@ fi
 
 set -u
 
-sed -i '' -e "s|${CURRENT_VERSION}|${NEW_VERSION}|" $(find . -name "README.md" -o -name "gradle.properties" | xargs -n 1)
+sed -i '' "s|${CURRENT_VERSION}|${NEW_VERSION}|" $(find . -name "README.md" -o -name "gradle.properties" | xargs -n 1)
```

Comment:
hmm I think this will break on mac with this change what OS are you on?

Reply:
mac

Reply:
tested your change works fine for me.
unsure why its needed but 👍